src/platforms/javascript/guides/nextjs/manual-setup/sentry-properties.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Sentry Properties
+sidebar_order: 2
+description: "Learn about alternative ways to set Sentry properties for your application."
+---
+
+If you can’t (or prefer not to) run the [configuration step](/platforms/javascript/guides/nextjs/#configure) when installing the SDK, you can follow the instructions below to configure your application.
+
+## Properties File
+
+You can create a `sentry.properties` file in the base directory of your project, as in the [configuration step](/platforms/javascript/guides/nextjs/#configure):
```

Comment:
Oh well, understanding that this file is not even required, why do we talk about it and then tell people to prefer environment variables?

Is there anything that can only be done in a property file?

The only apparent advantage is when this file is auto-generated with the wizard to get people started. Still it may not be advisable to store it in version control, specially when it has an auth token.

Reply:
Using environment variables is an alternative to this file, in case someone doesn't want to use it. Since the Wizard is generating it, I decided to include it in the docs; do you think it should be removed?